go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	golang.org/x/exp v0.0.0-20240409090435-93d18d7e34b8
 	golang.org/x/net v0.39.0
 	golang.org/x/oauth2 v0.29.0
-	google.golang.org/api v0.229.0
```

Comment:
this looked good but can you just resolve the conflicts that were introduced recently?